jena-fuseki2/jena-fuseki-main/src/test/java/org/apache/jena/fuseki/main/TestFusekiServerBuild.java

```diff
@@ -412,6 +415,26 @@ public class TestFusekiServerBuild {
         } finally { server.stop(); }
     }
 
```

Comment:
Would be useful to have an explicit test of setting `/` as the context path, while that is the implicit default someone will explicitly try to set it at some point

Reply:
Test added ... with some sidetracking into clearing up testing (I found some tests passed for the wrong reasons).